charon/acheron_sql.py

```diff
@@ -361,88 +353,87 @@ def get_charon_sample(self, sampleid):
         else:
             raise Exception('A connection error "{}" occurred while getting the sample from Charon'.format(r.status_code))
 
+
+    def update_charon_modifications(self, cur_doc, new_doc, url, session, headers):
+        keys_to_check = new_doc.keys()
+        if are_dicts_different(keys_to_check, cur_doc, new_doc):
+            merged = merge(cur_doc, new_doc)
+            if merged != cur_doc:
```

Comment:
Do we have a real case in mind where the dicts are different but the merged and cur_doc are not?

Reply:
That would depend on the merge function, which has not been touched in this PR(so far 😄)

Reply:
True